docs/performance/one-page.md

```diff
@@ -309,6 +309,46 @@ The following values can be used with the `-m` parameter of either the `run` or
 |`timeToFirstInteractive`|The amount of time it takes for a page to be able to reliably respond to user input.|seconds|
 |`totalBlockingTime`|The amount of time that elapses between `firstContentfulPaint` and `timeToFirstInteractive`, which is a key indicator of lag.|seconds|
 
+Detailed metric values can be found in performance.json.
+This file is generated when speedo is run with `-l` parameter.<br />
```

Comment:
Does the `-l` parameter also work with the`analyze` command?

Reply:
@lysannep yes, the `-l` parameter works with `analyze` command